feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -117,6 +125,21 @@ class IdentityManager(
         }
     }
 
+    private fun invalidateCustomerInfoAndETagCacheIfNeeded(appUserID: String) {
+        val cachedCustomerInfo = deviceCache.getCachedCustomerInfo(appUserID)
+        if (shouldInvalidateCustomerInfoAndETagCache(cachedCustomerInfo)) {
+            log(LogIntent.USER, IdentityStrings.INVALIDATING_CACHED_CUSTOMER_INFO)
```

Comment:
This is `debug` level. I wonder if this should be `info`, to make it more prominent... I thought it might be enough as `debug` though in the end.

Reply:
I think `info` would be useful actually 👍🏻